### DISCUSSION TITLE
I cannot disable battery module in starship

### DISCUSSION THREAD

**Discussion by morph-k** [starship/starship#3465] [Q&A]
I want to disable the battery module completely but I cannot get it to work in my starship.toml configuration file. I have tried without and with the quotes for setting the disabled field to true. Please help.

![Screen Shot 2022-01-19 at 9 55 30 AM](https://user-images.githubusercontent.com/23323356/150155974-9300e8d6-b3e6-4f72-953f-b0db726a86d4.png)
![Screen Shot 2022-01-19 at 9 57 20 AM](https://user-images.githubusercontent.com/23323356/150156083-7151bd3b-d054-4e3a-b541-c75190aa3a9d.png)

**Comment by chipbuster** [comment]
Have you configured starship to use the new configuration file? It looks like (I can't be certain) that your configuration file is at `~/.config/starship/starship.toml`, which won't be picked up by default.

**Reply by morph-k** [reply]
How do you configure starship to look at the starship.toml file

**Reply by chipbuster** [reply]
Starship will look at the `STARSHIP_CONFIG` environmental variable to determine where the config file is. You can set that variable to the path to your config file, e.g. `export STARSHIP_CONFIG=/path/to/config`.

**Reply by matchai** [reply]
It's also worth noting that Starship will automatically use `~/.config/starship.toml` (as opposed to `~/.config/starship/starship.toml`, as you had used).
